tests/dbgp/dbgpclient.php

```diff
@@ -1,16 +1,53 @@
 <?php
 class DebugClient
 {
+	protected $triedPorts = 0;
+
+	protected $startPort = 9991;
+
 	private $tmpDir;
 
+	protected function getPort()
+	{
+		return $this->startPort + $this->triedPorts;
+	}
+
+	protected function getIPAddress()
+	{
+		$records = dns_get_record(php_uname('n'), DNS_A);
```

Comment:
Again can't really remember if I wrote this because it was needed or some other reason. The latest version in the master uses 0.0.0.0 and not 127.0.0.1. Maybe 0.0.0.0 for IPv4 and ::1 for IPv6 is enough. I can make this change and see if there is any problems.